src/Compute/Compute/help/New-AzVMConfig.md

```diff
@@ -52,7 +52,7 @@ See [Quickstart: Create a Windows virtual machine in Azure with PowerShell](http
 ### Example 1: Create a virtual machine object
 ```powershell
 $AvailabilitySet = Get-AzAvailabilitySet -ResourceGroupName "ResourceGroup11" -Name "AvailabilitySet03"
-$VirtualMachine = New-AzVMConfig -VMName "VirtualMachine07" -VMSize "Standard_A1" -AvailabilitySetID $AvailabilitySet.Id
+$VirtualMachine = New-AzVMConfig -VMName "VirtualMachine07" -VMSize "Standard_A1" -AvailabilitySetID $AvailabilitySet.Id -Zone "1"
```

Comment:
>"1" [](http://example.com/codeflow?start=127&length=3)

Try to be consistent with the other sample.

Reply:
inconsistency on purpose to demonstrate different ways the input can be provided